pkg/providers/vsphere/placement/group_placement.go

```diff
@@ -23,7 +23,7 @@ func GroupPlacement(
 	namespace, childRPName string,
 	configSpecs []vimtypes.VirtualMachineConfigSpec) (map[string]Result, error) {
 
-	candidates, resourcePoolToZoneName, err := getPlacementCandidates(ctx, client, vcClient, "", namespace, childRPName)
+	candidates, err := getPlacementCandidates(ctx, client, vcClient, "", namespace, childRPName)
 	if err != nil {
 		return nil, err
```

Comment:
While this has not changed, is it worth taking this opportunity to make this a wrapped error so we can tell from where this is returned? Or are the inner errors clear enough?